schemas/pioneer-citp-news-disinfo/measurements/measurements.1.schema.json

```diff
@@ -148,21 +136,15 @@
                     "audience": {
                       "enum": [
                         "public",
-                        "restricted"
+                        "restricted",
+                        "unknown"
                       ],
                       "type": "string"
                     },
                     "classification": {
-                      "enum": [
-                        "pol news",
-                        "nonpol news",
-                        "other"
-                      ],
-                      "type": "string"
+                      "type": "integer"
```

Comment:
This type change won't play well with automated checks in our deployment pipeline. There are two ways to change this. The first is document this field as deprecated in a description and to add a new property (e.g. `pageClassification`) which would be evolution compatible. The other is to copy the schema with the version bumped.